mc/nat.go

```diff
@@ -119,7 +119,7 @@ func NATConfigFromString(str string) (cfg NATConfig, err error) {
 }
 
 func GetPublicIP() (string, error) {
-	res, err := http.Get("http://ifconfig.me/ip")
+	res, err := http.Get("http://ifconfig.co/ip")
```

Comment:
Sure, that's probably a good idea.
The code running ifconfig.co is available at github: https://github.com/martinp/ipd
So we can run it at `ifconfig.medichain.io` and have fallback to `ifconfig.co`.

I'll open an enhancement issue and add support once we have the ifconfing up.